drivers/stepper/gpio_stepper_controller.c

```diff
@@ -77,6 +77,21 @@ static void decrement_coil_charge(const struct device *dev)
 	}
 }
 
+static int power_down_coils(const struct device *dev)
+{
+	const struct gpio_stepper_config *config = dev->config;
+
+	for (int i = 0; i < NUM_CONTROL_PINS; i++) {
+		const int err = gpio_pin_set_dt(&config->control_pins[i], GPIO_ACTIVE_LOW);
```

Comment:
`GPIO_ACTIVE_LOW` is actually 1 `#define GPIO_ACTIVE_LOW         (1 << 0)`, its used to configure the polarity of the pin, you should simply pass 0